recipes/goss/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "goss" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/p/pygoss/pygoss-{{ version }}.tar.gz"
+  sha256: bb63b5f135b11f3366b92c3bc24c735f57c503ff6d04bb7de25ebdf053bd5d55
+
+build:
+  number: 1
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - pybind11
+    - ninja
+    - scikit-build
+    - cmake
+    - libgomp    # [linux]
+    - llvm-openmp     # [osx]
```

Comment:
Please read our documentation regarding openmp. OpenMP dependencies should only appear in the requirements/build section.

https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Reply:
Thanks for pointing that out. This should now be fixed in https://github.com/conda-forge/staged-recipes/pull/22029/commits/0963ef5f9636fa5befd203cfee4715d3ff8618d8